src/Chainweb/Version.hs

```diff
@@ -305,6 +305,9 @@ data Upgrade = Upgrade
     deriving stock (Generic, Eq)
     deriving anyclass (NFData)
 
+instance Show Upgrade where
+    show _ = "<upgrade>"
```

Comment:
why can (or should) this not derive Show?

Reply:
Upgrade transactions are *massive blobs* of base64-encoded text and are probably not useful. Maybe we should actually label them with names instead to get a decent show...